### PR TITLE
add Benqi Comptroller

### PR DESCRIPTION
**Pull request by ericruleman**
Adds all 18 events from the Benqi Comptroller to blockchain-etl.

Generated via https://nansen-contract-parser-prod.web.app/ on address `0xd8E426C61b0FBBda06e9F603263ABea09d717dBD` on Avalanche chain.

### CODE REVIEW

**Comment by ericruleman** [comment]
Used Contract Parser: https://nansen-contract-parser-prod.web.app/ @medvedev1088

File: dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+            ],
+            "name": "ActionPaused",
+            "type": "event"
+        },
+        "contract_address": "0xd8e426c61b0fbbda06e9f603263abea09d717dbd",
+        "field_mapping": {},
+        "type": "log"
+    },
+    "table": {
+        "dataset_name": "<INSERT_DATASET_NAME>",
```
Comment (tuanyuan2008) [inline dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json:33]: you should change the dataset names in the tables

File: dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json [hunk 2]
```diff
@@ -30,7 +30,7 @@
         "type": "log"
     },
     "table": {
-        "dataset_name": "<INSERT_DATASET_NAME>",
+        "dataset_name": "avalanche_benqi",
```
Comment (medvedev1088) [inline dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json:33]: Don't need the chain prefix `avalanche_`. Is this contract deployed on Avalanche?

Comment (tuanyuan2008) [reply dags/resources/stages/parse/table_definitions/benqi/Comptroller_event_ActionPaused.json:33]: Yea -- think this code might need to go in [this repo](https://github.com/nansen-ai/evmchain-etl-table-definitions/tree/main/parse/table_definitions_avalanche)

**Comment by ericruleman** [comment]
Closing this PR b/c it should actually be in the other repo — [link to new PR](https://github.com/nansen-ai/evmchain-etl-table-definitions/pull/58).

**Pull request closed without merging**